src/chi/components/header/header.scss

```diff
@@ -224,7 +224,13 @@
 }
 
 /* IE11 CSS hack */
-@media all and (-ms-high-contrast:none) {
+@media only screen and (max-width: $lg-breakpoint - 1) and (-ms-high-contrast:none) {
```

Comment:
@davidojedacl this is how I see in IE11. We should make it just for sm or md devices

![Screen Shot 2019-05-28 at 4 13 14 PM](https://user-images.githubusercontent.com/17043903/58485099-aa467380-8163-11e9-9726-9c7185f161a7.png)

![Screen Shot 2019-05-28 at 4 12 25 PM](https://user-images.githubusercontent.com/17043903/58485111-b0d4eb00-8163-11e9-9a17-1c1bf3ed7071.png)

Reply:
done